Sources/PubNub/PubNubConfiguration.swift

```diff
@@ -94,7 +93,7 @@ public struct PubNubConfiguration: Hashable {
     requestMessageCountThreshold: UInt = 100,
     filterExpression: String? = nil,
     enableEventEngine: Bool = false,
-    maintainPresenceState: Bool = false
```

Comment:
Initially, we've sent state with a heartbeat and subscribe? If yes, then `true` is the correct one, if not - `false`. This property needed not to break any user code which may not expect state to be set.